### PR TITLE
Feature/SC-3638 Filter UI changes

### PR DESCRIPTION
**Pull request by luizzaly**
# Issue: [SC-](https://ticketsystem.schul-cloud.org/browse/SC-3638)

## Description
Adjusted the design of the DataFilter

## Screenshot

Screenshot 2020-03-31 at 13 38 05
Screenshot 2020-03-31 at 13 38 29

### CODE REVIEW

**Comment by dominikbruechner** [comment]
irgendwas ist noch im storybook schepps 
Screen Shot 2020-03-31 at 17 10 17

**Comment by adrianjost** [comment]
> * see below, there is no mouse pointer visible when hovering the X
> * Furthermore the touch area around the X is too small, it should be bigger, as there is a second action with direct click on the chip.
> 
> Screen Shot 2020-03-31 at 17 04 32

It should also be keyboard focusable

**Comment by adrianjost** [comment]
> irgendwas ist noch im storybook schepps
> Screen Shot 2020-03-31 at 17 10 17

Das kann auch an der Lib liegen. Das war vorher schon so. Muss ich ggf. noch fixen

File: src/components/organisms/DataFilter/DataFilterModal.vue [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+		<template v-slot:header>
+			{{ title }}
+		</template>
+		<template v-slot:body>
+			<slot />
+		</template>
+		<template v-slot:footer>
+			<modal-footer>
+				<template v-slot:right>
+					<base-button design="text" @click="$emit('cancle')">
```
[the remark this answers is not in the document]
Comment (luizzaly) [reply src/components/organisms/DataFilter/DataFilterModal.vue:12]: Adrian will change that in the library

File: src/components/organisms/DataFilter/DataFilterModal.vue [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+						{{ labelCancle }}
```
[the remark this answers is not in the document]
Comment (luizzaly) [reply src/components/organisms/DataFilter/DataFilterModal.vue:13]: Adrian will change that in the library

File: src/components/organisms/DataFilter/DataFilterModal.vue [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+import ModalFooter from "@components/molecules/ModalFooter";
+
+export default {
+	components: {
+		ModalFooter,
+	},
+	props: {
+		title: { type: String, required: true },
+		labelApply: { type: String, required: true },
+		labelCancle: { type: String, required: true },
```
[the remark this answers is not in the document]
Comment (luizzaly) [reply src/components/organisms/DataFilter/DataFilterModal.vue:34]: same here :)

**Comment by luizzaly** [comment]
> * see below, there is no mouse pointer visible when hovering the X
> * Furthermore the touch area around the X is too small, it should be bigger, as there is a second action with direct click on the chip.
> 
> Screen Shot 2020-03-31 at 17 04 32

Screenshot 2020-04-01 at 17 14 51
Screenshot 2020-04-01 at 17 14 59

File: src/components/organisms/DataFilter/DataFilterModal.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (adrianjost) [reply src/components/organisms/DataFilter/DataFilterModal.vue:12]: sorry, the lib currently missspells those words, so we need to missspell it here as well :(
I will change that soon! @josafatburmeister already informed me about this issue.

**Pull request merged**